### ISSUE TITLE
Firebase version has upgraded to 9.2.0. Please release upgraded version using 9.2.0

### ISSUE THREAD

**Issue by gouravd** [firebase/FirebaseUI-Android#208]
If i use 9.2.0 in my app, then on using Firebase-UI (which uses 9.0.2) I get the following

FATAL EXCEPTION: main
                                                                        Process: XXXXXXXXX, PID: 4362
                                                                        java.lang.NoSuchMethodError: No virtual method zztw()Ljava/util/Set; in class Lcom/google/android/gms/common/internal/zzg; or its super classes (declaration of 'com.google.android.gms.common.internal.zzg' appears in /data/app/XXXXXXXXX-2/base.apk)

**Comment by wscheffer** [comment]
We really need this cant use this library with play services 9.2.0

**Comment by kirtan403** [comment]
I don't know why, but this stackoverflow [question](https://stackoverflow.com/questions/38156358/firebaseui-auth-is-crashing-on-android) didn't solve this issue. Please update the to 9.2.0

**Comment by samtstern** [comment]
@gouravd while we're working on the next version, if you want to use Firebase 9.2.0 libraries with FirebaseUI-Android, try this in your `build.gradle` file:

```
android {
    // ...

    configurations.all {
        resolutionStrategy.force 'com.google.firebase:firebase-auth:9.2.0'
        resolutionStrategy.force 'com.google.android.gms:play-services-auth:9.2.0'
    }
}

```

I think that will resolve your issue for now.

**Comment by kirtan403** [comment]
@samtstern When is the new build coming with firebase 9.2.0 ?

**Comment by samtstern** [comment]
@kirtan403 I just went ahead and released version `0.4.2` which should solve this issue by upgrading the underlying dependency.

**Comment by kirtan403** [comment]
@samtstern Awesome!
